src/lib/select/select.spec.ts

```diff
@@ -1042,10 +1042,14 @@ describe('MdSelect', () => {
       // Extra trigger height beyond the font size caused by the fact that the line-height is
       // greater than 1em.
       const triggerExtraLineSpaceAbove = (1 - triggerLineHeightEm) * triggerFontSize / 2;
+      const topDifference = Math.floor(optionTop) -
+          Math.floor(triggerTop - triggerFontSize - triggerExtraLineSpaceAbove);
 
-      expect(Math.floor(optionTop))
-          .toBe(Math.floor(triggerTop - triggerFontSize - triggerExtraLineSpaceAbove),
-              `Expected trigger to align with option ${index}.`);
+      // Expect the coordinates to be within a pixel of each other. We can't rely on comparing
+      // the exact value, because different browsers report the various sizes with slight (< 1px)
+      // deviations.
+      expect(Math.abs(topDifference) < 2)
```

Comment:
Could we make this a bit tighter - to 1 pixel difference? Concerned the test may not catch actual 1-2px offsets.

Reply:
This is 1 pixel though. `Math.abs(difference) < 2` will catch differences of 0 and 1.

Reply:
Oh, I missed the `Math.floor` above, so I was talking about differences of 1.5, etc. Ignore me then.